### PR TITLE
Add mark-boatman recommendations

### LINKED ISSUE #15: mark-boatman checklist
- [x] Create a branch
- [x] Add a file
- [ ] Make a PR
- [ ] Merge
- [ ] Done

### PR DESCRIPTION
**Pull request by mark-boatman**
Can you please check out my stellar changes?

Closes #15

### CODE REVIEW

File: mark-boatman.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# St Paul, MN
```
Comment (Bill-Watts) [inline mark-boatman.md:1]: @mark-boatman My first review comment

File: mark-boatman.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+
+### Places to Eat
+- Bag lunch
+- Arby's
+- Subway
```
Comment (Bill-Watts) [inline mark-boatman.md:6]: @mark-boatman My Second Review

**Review by Bill-Watts — CHANGES_REQUESTED** [review]

File: mark-boatman.md (hunk 2 above)
Comment (mark-boatman) [reply mark-boatman.md:6]: Hey @Bill-Watts  I made those changes.

**Review by Bill-Watts — APPROVED** [review]

**Comment by Bill-Watts** [comment]
@mark-boatman I approve of this code!

`String.equals`

**Pull request merged**